config/config.go

```diff
@@ -212,6 +230,12 @@ var (
 	DefaultExemplarsConfig = ExemplarsConfig{
 		MaxExemplars: 100000,
 	}
+
+	DefaultTracingConfig = TracingConfig{
+		ServiceName: "prometheus",
+		Endpoint:    "localhost:4317",
```

Comment:
The default should be empty, which means that tracing is completely disabled.

Reply:
Now that the tracing will be opt-in based on whether config is provided, I agree. I'll remove this.

Reply:
Done, defaults removed.